components/Badge/badge.html

```diff
@@ -0,0 +1,132 @@
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+            </section>
+    
+            <section>
+                <div class="components">
+                    <h3>Badge on different sized avatar</h3>
+                    <div class="preview-block">
+                        <div class="badge">
+                            <img class="avatar avatar-l" src="../../assets/human.png" alt="human">
+                            <span class="badge-status online"></span>
+                        </div>
+                        <div class="badge">
+                            <img class="avatar avatar-xl" src="../../assets/human.png" alt="human">
+                            <span class="badge-status offline"></span>
+                        </div>
+                        <div class="badge">
+                            <img class="avatar" src="../../assets/human.png" alt="human">
+                            <span class="badge-status idle"></span>
+                        </div>
+                        <div class="badge">
+                            <img class="avatar avatar-m" src="../../assets/human.png" alt="human">
+                            <span class="badge-status dnd"></span>
+                        </div>
+                    </div>
+                </div>
+                <div class="code-block">
+                    <iframe
+                        src="https://carbon.now.sh/embed/Jl3eRibIUrkeh1QLKwim"
+                        style="width: 685px; height: 414px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                  </iframe>
+                </div>
+            </section>
+    
+            <section>
+                <div class="component">
+                    <h3>Badge on Icon</h3>
+                    <div class="preview-block">
+                        <div class="badge">
+                            <img class="icon" src="../../assets/mail-icon.png" alt="mail">
+                            <span class="badge-count">1</span>
+                        </div>
+                        <div class="badge">
+                            <img class="icon" src="../../assets/cart-icon.png" alt="cart">
+                            <span class="badge-count">1</span>
+                        </div>
+                        <div class="badge">
+                            <img class="icon" src="../../assets/notification-icon.png" alt="notification">
+                            <span class="badge-count">1</span>
+                        </div>
+                    </div>
+                </div>
+                <div class="code-block">
+                    <iframe
+                        src="https://carbon.now.sh/embed/N3o6inF2OE7mdqU22kPp"
+                        style="width: 745px; height: 330px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
You could move the iframe styling to the common css file. This would help with maintenance and in keeping the html clean.

Reply:
noted!